src/vstest.console/vstest.console.csproj

```diff
@@ -22,6 +22,7 @@
       (e.g. System.Runtime 4.1.2 instead of 4.0.0)
     -->
     <AutoGenerateBindingRedirects>false</AutoGenerateBindingRedirects>
+    <DefineConstants>$(DefineConstants);FIX_NULLABLE_CONFLICTS</DefineConstants>
```

Comment:
Using this trick as some of the files that need to be fixed are included from a shared project that would force us to define the alias in all using projects.

Reply:
@cvpoienaru Re-reading that now, I am thinking that it could be clearer to use `USE_EXTERN_ALIAS` instead, WDYT?